runtime/runtime-params-estimator/src/utils.rs

```diff
@@ -267,3 +290,35 @@ pub(crate) fn generate_data_only_contract(data_size: usize) -> Vec<u8> {
     let wat_code = format!("(module (data \"{payload}\") (func (export \"main\")))");
     wat::parse_str(wat_code).unwrap()
 }
+
+#[cfg(test)]
+mod test {
+    use super::percentiles;
+    use crate::{config::GasMetric, gas_cost::GasCost};
+    use rand::prelude::SliceRandom;
+
+    #[track_caller]
+    fn check_percentiles(gas_values: &[u64], p_values: &[f32], expected_gas_results: Vec<u64>) {
```

Comment:
You could change the third argument to `&[u64]` and then you wouldn’t need to create the vectors.  It is a test so we don’t really care about performance but to me it feels cleaner regardless.